include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -239,24 +239,24 @@ class argument_parser
                     std::string const & long_id,
                     std::string const & desc,
                     option_spec const spec = option_spec::DEFAULT,
-                    validator_type validator = validator_type{}) // copy to bind rvalues
+                    validator_type verificator = validator_type{}) // copy to bind rvalues
```

Comment:
Verificator is not an actual word. It would be rather a `verifier`. But what is the semantic of the validator? It says it is invocable, but is it not specifically a predicate. The standard uses the term `predicate` to indicate that something is checked and then returns true/false depending on the result. I'd go for this name then.

Reply:
On a second thought, we want to have more functionality. So maybe we call it then `argument_validator` or `option_validator` then it is different to just the concept name but at the end it is a validator. Similar to the cppref calling a std::predicate `pred`, but better :)

Reply:
I like `option_validator` ! :)